### DISCUSSION TITLE
Is this compatible with Capacitor 3?

### DISCUSSION THREAD

**Discussion by AlbajramiKevin** [robingenz/capacitor-badge#42] [Q&A]
I get an [!] CocoaPods could not find compatible versions for pod
        "CapawesomeCapacitorBadge" error when trying to sync this app
Package.json 
"@capacitor-community/fcm": "^2.0.2",
    "@capacitor/android": "^3.5.0",
    "@capacitor/app": "1.1.1",
    "@capacitor/camera": "^1.3.1",
    "@capacitor/core": "^3.0.2",
    "@capacitor/ios": "^3.5.0",
    "@capacitor/push-notifications": "^1.0.9",
    "@capacitor/share": "1.0.3",
    "@capacitor/splash-screen": "^1.2.2",
    "@capawesome/capacitor-badge": "^2.0.0",

**Comment by robingenz** [comment]
You need to install '@robingenz/capacitor-badge@1.2.1'.
